guides/editorial-style-guide.md

```diff
@@ -49,7 +49,7 @@ Don't assume reader familiarity with crypto concepts. Use analogies and examples
 
 ## Tone and Voice
 
-”Although tone and voice are often used together, they are not at all the same thing. Voice is the overall **personality** of your brand and can be described in adjectives like helpful, witty, or friendly. \[…] your brand’s voice will not change. However, tone, or tone of voice, is the attitude of your writing for a particular content piece.” - Shelby Crawford
+> ”Although tone and voice are often used together, they are not at all the same thing. Voice is the overall **personality** of your brand and can be described in adjectives like helpful, witty, or friendly. \[…] your brand’s voice will not change. However, tone, or tone of voice, is the attitude of your writing for a particular content piece.” \~ Shelby Crawford
```

Comment:
We used `\~` below in citations, so I decided to use it here as well + I think it's better to use blockquote (or `>` in markdown) to outline citations

Reply:
We may also want to add a link to `Shelby Crawford`